src/main/java/seedu/dictionote/ui/MainWindow.java

```diff
@@ -296,7 +295,7 @@ private void configNoteSplit() {
     }
 
     /**
-     * Dynamically detect and change the split ratio depending on the dictionary and note visibility
+     * Detect and change the split ratio depending on the dictionary and note visibility.
```

Comment:
Shouldn't this be *"Detect**s**"* instead?

Reply:
fixed